src/ui/page-issue-link.tsx

```diff
@@ -8,7 +8,16 @@ export const PageIssueLink: Component = () => {
 	const i18n = useI18n();
 
 	const srcPath = createMemo(() => {
-		return `https://github.com/solidjs/solid-docs-next/issues/new?assignees=ladybluenotes&labels=improve+documentation%2Cpending+review&projects=&template=CONTENT.yml&title=[Content]:&subject=${getEntryFileName()}`;
+		return (
+			`https://github.com/solidjs/solid-docs-next/issues/new` +
+			`?assignees=ladybluenotes` +
+			`&labels=improve+documentation%2Cpending+review` +
+			`&projects=` +
+			`&template=CONTENT.yml` +
+			`&title=[Content]:` +
+			`&subject=${getEntryFileName()}` +
+			`&page=${window.location.href}`
+		);
 	});
```

Comment:
```suggestion
	const srcPath = createMemo(() => {
		return (
			"https://github.com/solidjs/solid-docs-next/issues/new" +
			"?assignees=ladybluenotes" +
			"&labels=improve+documentation%2Cpending+review" +
			"&projects=" +
			"&template=CONTENT.yml" +
			"&title=[Content]:" +
			`&subject=${getEntryFileName()}` +
			`&page=${window.location.href}`
		);
	});

```

This will resolve the error. I apologize for the back and forth. We are working on a CI step that will automatically format files, so you shouldn’t have to deal with this hassle in the future.